src/components/Person/Person.jsx

```diff
@@ -1 +1,40 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+const MALE = 'm';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const partnerStatus = (
+    sex === MALE
+      ? 'wife'
+      : 'husband'
+  );
```

Comment:
Outer brackets are redundant here 
```suggestion
  const partnerStatus = sex === MALE
      ? 'wife'
      : 'husband';
```